ios/RNSScreenStackHeaderSubview.mm

```diff
@@ -22,6 +23,21 @@ @implementation RNSScreenStackHeaderSubview
 
 #pragma mark - Common
 
+- (RNSScreenStackHeaderConfig *)getHeaderConfig
+{
+  RNSScreenStackHeaderConfig *headerConfig = (RNSScreenStackHeaderConfig *)self.reactSuperview;
+  return headerConfig;
+}
```

Comment:
```suggestion
- (nullable RNSScreenStackHeaderConfig *)getHeaderConfig
{
  return (RNSScreenStackHeaderConfig * _Nullable)self.reactSuperview;
}
```

Reply:
It might be even worth adding a check whether the `reactSuperview` is of appropriate type, e.g. in debug mode, using `NDEBUG` symbol and `isKindOfClass`

Reply:
https://github.com/software-mansion/react-native-screens/pull/2385/commits/0667d88d9a698627459506459b0094e281de58e0